css/properties/display.json

```diff
@@ -143,7 +143,7 @@
                 "version_added": "11.3"
               },
               "samsunginternet_android": {
-                "version_added": true
+                "version_added": "7.0"
```

Comment:
```suggestion
                "version_added": "9.0"
```

Reply:
`I think this came with CSS GRid in Version 7

Reply:
`display: contents` isn't a part of the CSS Grid spec (Level 1 or 2), but rather the CSS Display module.   [CanIUse also reports Samsung Internet 9](https://caniuse.com/#feat=css-display-contents) for this feature, so I'm not too certain about that?